src/views/Home.vue

```diff
@@ -34,7 +34,7 @@
           :title="section.name"
           :icon="section.icon || undefined"
           :displayData="getDisplayData(section)"
-          :groupId="`section-${index}`"
+          :groupId="`${pageId}-section-${index}`"
```

Comment:
This is because, now with multi-page support, we need to identify the difference between two sections with the same name + index, but on different pages.